harvest/api/dummy.py

```diff
@@ -112,10 +130,7 @@ def fetch_price_history(self,
             high.append(h)
             low.append(l)
             close.append(c)
-            volume.append(v)
-
-        if not symbol:
-            symbol = 'DUMMY'            
```

Comment:
this is not ever hit because `fetch_price_history` requires users to input a stock value.